src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -0,0 +1,381 @@
+| PEAKS                | Directions of ODF maxima on the sphere.                                                                                                                                                                                                                                                                              |
+
+## Describing tractography
+
+Tractography normally generates one of two primary file types: tractograms or
+NIfTi files containing maps. Tractograms are files containing a collection of
+streamlines, which are objects describing the path of idealized brain fiber
+fascicles (ensembles of neural fibers that travel together). Tractography maps
+contain visitation counts (i.e. 3D spatial histograms) of a collection of
+streamlines but do not contain the actual streamline objects. The main type of
+map represent the diffusion path probability, or how likely it is that diffusion
+between pairs of brain regions. Two tractogram file formats are supported,
+tractography maps should be saved as NIfTIs.
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>[_referencemap-<referencemap>][_desc-<label>][_subset-<label>]_tractography.[trk|tck|nii[.gz]]
+```
+
+### Filename keys
+
+`desc` (optional) – A way to refer to a specific instance of the tractography
+process. The combination of tractography methods and parameters that were used
+to create this tractography result should be described in the associated JSON
+file.
+
+`subset` (optional) – A label descriptor for the the subset of streamlines
+included in this file; if not specified, a whole brain tractography is assumed
+(a tractogram). Example of subsets can be `short` if only short streamlines were
+kept, or `subsampled50` if only 50% of the streamlines were kept.
+
+### File formats
+
+- [.trk - Diffusion Toolkit (+TrackVis)](http://www.trackvis.org/docs/?subsect=fileformatEmbeds)
+- [.tck - MRTrix](http://mrtrix.readthedocs.io/en/latest/getting_started/image_data.html#tracks-file-format-tckAssumes)
+
+### JSON Metadata
+
+We have two main classes of tractography algorithms: Global or Local
+tractography, each with various supported algorithms.
+
+| Class  | Algorithms | Reference                                                                                                                                                       |
+| ------ | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| LOCAL  | PROB       | Sherbondy, A.J., et al. (2008).Tournier J.D.et al. (2012).Behrens et al. (2007)                                                                                 |

```

Comment:
=> "Sherbondy, A.J., et al. (2008), Tournier J.D.et al. (2012), Behrens et al. (2007)" 

Comma and space, instead of period and no space.